files/__addonLocation__/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": "@tsconfig/ember/tsconfig.json",
+  "compilerOptions": {
+    "paths": {
+      // No paths, no absolute imports, only node_modules imports allowed
+      // But fallback for type-overrides and such
+      "*": ["types/*"]
+    }
+  },
+  "include": [
+    "src/**/*",
+    "types/**/*"
+  ],
+  "glint": {
+    "environment": "ember-loose",
+    "transform": {
+      "include": ["src/**"]
+    }
```

Comment:
```suggestion
```

Good news: there should basically never be a reason to specify a `transform` key in Glint config at this point. It was originally meant as an escape hatch in case we had bugs or performance issues in Glint itself that would otherwise make it impossible to use in a large project. Later on, it became an ad-hoc way of doing incremental adoption, but it was never ideal since `include`/`exclude` opts files out of Glint analysis and editor support _entirely_, not just typechecking.

Today, `@glint-nocheck` directives are a better way of incrementally adopting Glint (particularly once we land [automation for it](https://github.com/typed-ember/glint/pull/427)), though I suspect I'll still be stamping out `transform` usage for a long time yet to come, as the pattern seems to be quite persistent in the community 😅

Reply:
Got it, removed.